osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/application/config/AbstractMessagingConfig.java

```diff
@@ -25,6 +25,7 @@
 /**
  * This class provides the basic components used for JMS messaging.
  */
+@Deprecated
 public abstract class AbstractMessagingConfig extends AbstractConfig {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1133)